src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyErrorBoundaryLogger.cs

```diff
@@ -21,7 +19,7 @@ public ValueTask LogErrorAsync(Exception exception)
     {
         // For, client-side code, all internal state is visible to the end user. We can just
         // log directly to the console.
-        _errorBoundaryLogger.LogError(exception.ToString());
+        _errorBoundaryLogger.LogError(exception.ToString(), exception);
```

Comment:
I think this is wrong, as the `exception` comes before the `message`. See #39223